app/models/lettings_log.rb

```diff
@@ -21,6 +21,8 @@ class LettingsLog < Log
   include Validations::DateValidations
   include Validations::FinancialValidations
 
+  default_scope { where(visible: true) }
```

Comment:
This probably worth a discussion! 😬 

I'm generally not of default_scope because it impacts initialization amongst other funky things. e.g. 

```ruby
LettingsLog.new.visible # => true

scheme = Scheme.last
scheme.lettings_logs.destroy_all # Only deletes visible logs
```

Thoughts?

Reply:
Did you consider adding another state to `Log#status` that is before `not_started` ?

Reply:
Yeah i already considered this and not also not a fan of `default_scope`. the alternative is to add explicitly add the scope `visible` to pretty much every existing AR thing that involves lettings logs. Thats the only alternative i can think of.

Reply:
I don't think it will be too difficult. Searching for `lettings_logs.*` and `LettingsLog.` doesn't yield that many results. I suggested using `status` because it's more likely you won't have to make as any many changes, because some places scope out `in_progress` or `completed` specifically.